tests/unit/module_utils/test_linode_common_info.py

```diff
@@ -9,7 +9,7 @@
 
 class TestLinodeInfoModule:
 
-    @pytest.fixture
+    @pytest.fixture(scope="function")
```

Comment:
Added the function scope here just for some extra confidence that the below tests will not conflict.